test-integration/helpers/ui/user.coffee

```diff
@@ -94,7 +94,7 @@ class User extends TestHelper
     @el.loginSubmit().click()
 
   login: (username, password = 'password') =>
-    @el.loginLink().isDisplayed().then (isDisplayed) =>
```

Comment:
Hmm, ok. I guess I didn't realize that difference between `.isDisplayed` and `.get().isDisplayed`. Maybe @openstax/test will run into the same problem?